src/library/Uno.Toolkit.Material/Styles/Controls/v2/TabBar.xaml

```diff
@@ -989,7 +991,7 @@
                                 <VisualState x:Name="ContentOnly">
                                     <VisualState.Setters>
                                         <Setter Target="IconBox.Visibility" Value="Collapsed" />
-                                        <Setter Target="ContentPresenter.Margin" Value="12,0" />
+										<Setter Target="ContentPresenter.Margin" Value="{ThemeResource TopTabBarItemContentOnlyMargin}" />
```

Comment:
I like that we are moving this to a resource but it won't cover all cases. If you have a large enough string, it'll still clip. So we should probably add a TextTrimming="CharacterEllipsis" or something to the contentpresenter, if that's available